src/Database/Query.php

```diff
@@ -295,6 +295,10 @@ public function select($select)
      */
     public function join(string $table, string $on, string $type = 'JOIN')
     {
+        if (strtolower($type) != 'join') {
+            $type .= ' join';
+        }
```

Comment:
I would prefer like following to support different uses as `xxx join` (already includes `join` word) with `join()`

```suggestion
        if (strpos(strtolower($type), 'join') === false) {
            $type .= ' join';
        }
```